app/models/organisation.rb

```diff
@@ -186,6 +186,7 @@ def foi_contacts
 
   before_destroy { |r| r.destroyable? }
   after_save :ensure_analytics_identifier
+  after_commit :publish_services_and_information_content_item
```

Comment:
We should use a Sidekiq worker for this. It'll be more pleasant for the user updating the organisation and will add automatic retry-ing when the call to publishing-api fails.

Would it be possible to move this to the controller by the way? Callbacks for external side effects aren't good.